src/Uno.UI/UI/LayoutHelper.cs

```diff
@@ -93,53 +93,41 @@ internal static Size GetMarginSize(this IFrameworkElement frameworkElement)
 		}
 
 		[Pure]
-		internal static (Point offset, bool overflow) GetAlignmentOffset(this IFrameworkElement e, Size clientSize, Size renderSize)
+		internal static Point GetAlignmentOffset(this IFrameworkElement e, Size clientSize, Size renderSize)
 		{
-			// Start with Bottom-Right alignment, multiply by 0/0.5/1 for Top-Left/Center/Bottom-Right alignment
```

Comment:
The changes done here are to reduce unnecessary calculations. Previously, we would calculate the offset as bottom-right, then we throw away the calculation if we are aligning  horizontally to left or vertically to top. Now, the code is more straightforward.